### PR TITLE
Km gvs add storage cost and cleanup doc

### PR DESCRIPTION
**Pull request by kayleemathews**

### CODE REVIEW

**Comment by koncheto-broad** [comment]
Seems reasonable.  Why was the license info removed?

**Comment by kbergin** [comment]
LGTM except plus one wondering why the license was removed. I imagine it should stay?

File: scripts/variantstore/beta_docs/gvs-quickstart.md [hunk 1]
```diff
@@ -148,8 +156,4 @@ If you use plan to publish data analyzed using the GVS workflow, please cite the
 
 Details on citing Terra workspaces can be found here: [How to cite Terra](https://support.terra.bio/hc/en-us/articles/360035343652)
 
-Data Sciences Platform, Broad Institute (*Year, Month Day that the workspace was last modified*) gvs-prod/Genomic_Variant_Store_Beta [workspace] Retrieved *Month Day, Year that workspace was retrieved*, https://app.terra.bio/#workspaces/gvs-prod/Genomic_Variant_Store_Beta
-
-### License
-**Copyright Broad Institute, 2022 | BSD-3**
```
Comment (kbergin) [inline scripts/variantstore/beta_docs/gvs-quickstart.md:154]: Why is this removed?

**Review by kbergin — CHANGES_REQUESTED** [review]

File: scripts/variantstore/beta_docs/gvs-quickstart.md (hunk 1 above)
Comment (kayleemathews) [reply scripts/variantstore/beta_docs/gvs-quickstart.md:154]: Removed because the license info was for WARP

Comment (kayleemathews) [reply scripts/variantstore/beta_docs/gvs-quickstart.md:154]: Is there something we should replace that with?

Comment (kbergin) [reply scripts/variantstore/beta_docs/gvs-quickstart.md:154]: Technically bc the code is in GATK repo it's currently under that license: https://github.com/broadinstitute/gatk/blob/master/LICENSE.TXT

Let's use that for now.

Comment (kayleemathews) [reply scripts/variantstore/beta_docs/gvs-quickstart.md:154]: Added GATK license

File: scripts/variantstore/beta_docs/gvs-overview.md [hunk 2]
```diff
@@ -124,6 +124,10 @@ Details on citing Terra workspaces can be found here: [How to cite Terra](https:
 
 Data Sciences Platform, Broad Institute (*Year, Month Day that the workspace was last modified*) gvs-prod/Genomic_Variant_Store_Beta [workspace] Retrieved *Month Day, Year that workspace was retrieved*, https://app.terra.bio/#workspaces/gvs-prod/Genomic_Variant_Store_Beta
 
+### License
+**Copyright Broad Institute, 2021 | Apache**
```
Comment (kbergin) [inline scripts/variantstore/beta_docs/gvs-overview.md:128]:
```suggestion
**Copyright Broad Institute, 2022 | Apache**
```

I just update the year on licenses whenever I see they're wrong.

**Review by kbergin — APPROVED** [review]
